druid/README.MD

```diff
@@ -0,0 +1,54 @@
+# Druid
+
+This script installs Druid on dataproc clusters. To learn more about Druid see documentation: http://druid.io/docs/latest/design/.
+
+
+## Using this initialization action
+You can use this initialization action to create a new Dataproc cluster with Druid installed.
+
+Once the cluster has been created, Druid Index Coordinator Console is configured to run on port `8090`. You can change this port with additional parameter:
```

Comment:
Druid Console runs on port 8081 - may be that is more relevant here.

Reply:
Modified README with this:
`Once the cluster has been created, Druid Overlord Console is configured to run on port 8090. You can change this port with additional parameter: --metadata=druid-overlord-port=xxxx Druid Coordinator Console is configured to run on port 8081. You can change this port with additional parameter: --metadata=druid-coordinator-port=xxxx`